components/ValueNFT/index.tsx

```diff
@@ -87,17 +87,16 @@ const ValueNFT: React.FC<Props> = (props: Props) => {
   }
 
   const updateMaturityDate = async () => {
+    if (isTinlakeV2(props.tinlake)) throw Error('Tinlake V2 does not have maturity date')
```

Comment:
Does this work? I always do `throw new Error()`. Isn't that a class you have to instantiate?

Reply:
Yes you are right. Weird that there's no type error.

Reply:
Fixed